content/800-data-platform/050-data-proxy/index.mdx

```diff
@@ -0,0 +1,103 @@
+
+Data Proxy infrastructure is currently available in two regions:
+
+* US East 1 (West Virginia, United States)
+* EU Central 1 (Frankfurt, Germany)
+
+<Admonition>
+
+**Request a new Data Proxy region**<br /><br />
+
+If you want us to make Data Proxy available in a new region, you can submit your request [here](
+https://prisma103696.typeform.com/to/Y03RBE5c).
+
+</Admonition>
+
+In each region, the Data Proxy infrastructure has multiple clusters that enable the load balancing, high availability, and failover of database connection pooling and management.
+
+You can use Data Proxy with [all databases that Prisma ORM supports](/reference/database-reference/supported-databases).
+
+## Connection pooling and connection management
+
+Data Proxy provides connection pooling so that your application can reuse active connections and never exhaust the maximum number of connections that your database allows.
+
+Connection pooling with the Data Proxy utilizes orchestrated Prisma [query engine](/concepts/components/prisma-engines/query-engine) technology to open, maintain, and close connection pools and, thus, help applications load-balance and scale database requests based on demand.
+
+A Data Proxy connection pool maintains active database connections and ensures that an application reuses those when new requests come in.
+
+In each region, Data Proxy management services orchestrate the lifecycle of connection pools. The lifecycle of a Data Proxy connection pool goes through the following stages:
+
+* the assignment of connection pools from each available cluster in the region
+* the increase of connection pools when the workload of your application increases
+* the maintenance of idle connection pools
+* the decrease and stop of connection pools after six minutes of inactivity within a pool
+
+This orchestration ensures that you accumulate Data Proxy usage (in CPU milliseconds) only when your database processes queries.
+
+The orchestration of Data Proxy connection pools ensures high availability, load balancing, and scalability of your application.
+
+* **High availability**. In each region, the Data Proxy infrastructure runs multiple clusters. By default and until your application reaches higher workloads, from each cluster the Data Proxy starts and assigns a connection pool to an application. If a cluster goes through a period of downtime, the remaining ones take over the management of connection pools.
+    <Admonition>
+
+    **Note**<br /><br />
+
+    You can select only one of the available regions for an environment of your application in the Prisma Data Platform.
+
+    Because of this, the high availability of Data Proxy infrastructure applies within a single region. Data Proxy connection pools from other regions cannot take over an application's connection pool from the current region.
```

Comment:
```suggestion
    Because of this, the high availability of Data Proxy infrastructure applies inside a single region. Data Proxy connection pools from other regions cannot take over an application's connection pool from the current region.
```